_posts/2020-02-05-cockpit-212.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 212 and Cockpit-podman 13
+author: Gundersanne
+date: 2020-02-05
+tags: cockpit linux podman
+slug: cockpit-212
+category: release
+summary: Cockpit with documentation and time localization improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 212.
+
+### Per page documentation
+
+The help menu now includes links to page- and OS-specific documentation.
+
+![Documentation per page](/images/documentation-per-page.png){:.screenshot}
+
+### Localize times
+
+The Overview, Accounts, and Logs pages now have localized time formats.
+
+![Localized time formats](/images/overview-localized-time.png){:.screenshot}
+
+### Podman: show full log
+
+Show containers' logs from when they were spawned.
```

Comment:
perhaps "since their startup, not just since opening the Logs tab."?

Reply:
+1